pages/dkp/konvoy/1.7/install/install-small-env/index.md

```diff
@@ -0,0 +1,570 @@
+---
+layout: layout.pug
+navigationTitle: Install on a small environment
+title: Install Konvoy on a small environment
+menuWeight: 40
+excerpt: Install Konvoy on a small environment
+beta: true
+enterprise: false
```

Comment:
@cmastr we could get rid of this PR by setting: `preview: true` here. just overcommunicating :)

Reply:
I think this was a backport issue from Konvoy. this is 1.7 docs which is not in beta. I think we need to fix it in Konvoy repo.